.github/pull_request_template.md

```diff
@@ -0,0 +1,24 @@
+Thank you for your pull request. Please fill the following information.
+
+---
+
+**Affects User Experience?** Yes/No
+
```

Comment:
Suggest rephrasing the question, and instead of putting "Yes/No" at the end of the question, can we render these options in a list where one or the other should be selected?  i.e., 

**Do changes in this PR impact User Experience (e.g., how the user interacts with Yo Office and/or the files and folders the user sees in the project that Yo Office creates)?**

- [ ] Yes

- [ ] No

Reply:
We can tell if there are any changes to the templates themselves, as the files under src\app\templates would be changed. I think we could simplify this to just ask if there are any changes to the Yo Office user experience.

**Do these changes impact how the user interacts with Yo Office?**

Reply:
> Suggest rephrasing the question, and instead of putting "Yes/No" at the end of the question, can we render these options in a list where one or the other should be selected? i.e.,
> 
> **Do changes in this PR impact User Experience (e.g., how the user interacts with Yo Office and/or the files and folders the user sees in the project that Yo Office creates)?**
> 
> * [ ]  Yes
> * [ ]  No

ah! cool, will do

Reply:
> We can tell if there are any changes to the templates themselves, as the files under src\app\templates would be changed. I think we could simplify this to just ask if there are any changes to the Yo Office user experience.
> 
> **Do these changes impact how the user interacts with Yo Office?**

cool, maybe will added that as a note: If you have files changed in the template folder, please mark "Yes"